test/backend/routes.spec.js

```diff
@@ -198,60 +223,192 @@ describe('Routes - ', () => {
         .catch(done);
     });
 
-    it('index page - returns 200', function(done) {
-        GET('')
-        .then(res => {
-            assert.equal(res.status, 200);
-            done();
-        })
+    it('allows access to login page without logging in', function(done) {
+        GET('login')
+        .then(() => done())
         .catch(done);
     });
 
-    // Settings
-    it('settings/urls - returns 200 and the urls', function(done) {
-        GET('settings/urls')
-        .then(res => res.json().then(json => {
-            assert.equal(res.status, 200);
-            assert.equal(json.http, 'http://localhost:9494');
-            assert.equal(json.https, '');
-            done();
-        }))
+    it('allows access to connections page without logging in', function(done) {
+        GET('')
+        .then(() => done())
```

Comment:
note that calling `GET` isn't enough to show that it's allowed as `fetch` doesn't raise an error for non-200 level status codes. You'll have to explicitly do `GET('').then(res => {assertEqual(res.status, 200); done()})`